parts/k8s/addons/coredns.yaml

```diff
@@ -100,6 +100,27 @@ spec:
       annotations:
         seccomp.security.alpha.kubernetes.io/pod: docker/default
     spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - podAffinityTerm:
```

Comment:
Let's try to summarize what this is expressing. Is it:

"Don't schedule coredns pods onto a node that already has a `kube-dns`-labeled pod in the same availability zone."

Reply:
Yes, except it will only try, if it can't it will schedule both pods on the same node if no other is available.

Reply:
Indeed, thanks. Hence the `preferredDuringSchedulingIgnoredDuringExecution` instead of  `requiredDuringSchedulingIgnoredDuringExecution`.

Reply:
Exactly